automation/check-patch.e2e-okd.sh

```diff
@@ -22,11 +22,12 @@ main() {
     trap teardown EXIT SIGINT SIGTERM SIGSTOP
     make cluster-sync
     make \
-        E2E_TEST_ARGS='-ginkgo.noColor -ginkgo.skip .*NNS.*cleanup.*|.*OVS.*' \
+        E2E_TEST_ARGS="$E2E_TEST_ARGS -ginkgo.skip .*NNS.*cleanup.*|.*OVS.*" \
         test/e2e
     make \
-        E2E_TEST_ARGS='-ginkgo.noColor -ginkgo.focus .*NNS.*cleanup.*' \
+        E2E_TEST_ARGS="$E2E_TEST_ARGS -ginkgo.focus .*NNS.*cleanup.*" \
```

Comment:
With this you overwrite previous junit output, don't you?

Reply:
You right, let's just have two different files, we have to follow the pattern at https://github.com/kubevirt/project-infra/blob/master/github/ci/prow/files/config.yaml#L28